### PR TITLE
Feat/criteria category creation

### PR DESCRIPTION
**Pull request by amundlrohne**
- Create new criteria categories

### CODE REVIEW

File: apps/frontend/src/app/containers/signUp/SignUp.module.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.submit {
```
Comment (perkynades) [inline apps/frontend/src/app/containers/signUp/SignUp.module.scss]: Rename :P

File: apps/frontend/src/app/containers/signUp/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const defaultUser: IUser = {
+    id: 0,
+    name: '',
+    email: '',
+    phone: '',
+    latestUpdate: Date.now().toString(),
+    criterias: []
+};
+
+export const SignUp = (): ReactElement => {
```
Comment (perkynades) [inline apps/frontend/src/app/containers/signUp/SignUp.tsx]: Rename

**Review by perkynades — APPROVED** [review]
Remmber for fix the signup page to register

File: apps/frontend/src/app/containers/signUp/SignUp.tsx (hunk 2 above)
Comment (amundlrohne) [reply apps/frontend/src/app/containers/signUp/SignUp.tsx]: Seems like the files were duplicated for some reason :P

**Review by perkynades — APPROVED** [review]

**Pull request merged**